applications/rag/frontend/README.md

```diff
@@ -3,98 +3,21 @@
 This directory contains the code for a frontend flask webserver integrating with the inference
 backend and (NYI) vector database through [LangChain](https://python.langchain.com/docs/get_started/introduction).
 
-## Cloud Run Setup
+The image is hosted on `us-central1-docker.pkg.dev/ai-on-gke/rag-on-gke` and used by the frontend deployment.
```

Comment:
Let's take out specifying which image is used. This'll be documented in the code.